wolfi-baselayout.yaml

```diff
@@ -37,7 +37,7 @@ pipeline:
 
   - name: Install
     runs: |
-      for i in bin etc etc/profile.d etc/secfixes.d home lib root var/log usr/bin usr/sbin usr/local tmp var/spool/cron opt run usr/lib; do
+      for i in bin etc etc/profile.d etc/secfixes.d home lib root var/log usr/bin usr/sbin usr/local/lib tmp var/spool/cron opt run usr/lib; do
```

Comment:
this kind of raises the question if /usr/local/lib should be in /etc/ld.so.conf.  
thoughts?

Reply:
it already is....

```console
# cat /etc/ld.so.conf 
/usr/local/lib
/usr/local/lib64
/lib
/lib64
/usr/lib
/usr/lib64
```

Reply:
given lib64 are symlinks, it is pointless for them to be there but hey ho =)

Reply:
oh yeah, i had forgotten that they were in there. duh. thanks.